tests/test_all.py

```diff
@@ -1,63 +1,257 @@
-"""
-Test that files that define __all__ aren't missing any exports.
+"""Test exported names"""
 
-You can add names that shouldn't be exported to _all_ignore, like
+import builtins
 
-_all_ignore = ['sys']
+import pytest
 
-This is preferable to del-ing the names as this will break any name that is
-used inside of a function. Note that names starting with an underscore are automatically ignored.
-"""
+from ._helpers import wrapped_libraries
 
+NAMES = {
```

Comment:
Maybe a better version of this test could automatically scrape `data-apis/array-api/`?

Reply:
cf https://github.com/data-apis/array-api-tests/blob/2db6c7b807a609a1539e312e01af093a45d34764/array_api_tests/test_has_names.py#L25
(however, https://github.com/data-apis/array-api-tests/issues/224)

Reply:
Are you suggesting that array-api-compat should have array-api as a git submodule, for testing only?
If so, do you agree that such a change is best left to a follow-up?

Reply:
> Are you suggesting that array-api-compat should add array-api as a git submodule, for testing only?

Definitely not.

> If so, do you agree that such a change is best left to a follow-up?

Absolutely yes.